Earthfile

```diff
@@ -52,6 +54,37 @@ test:
     RUN ginkgo run --fail-fast --slow-spec-threshold 30s --covermode=atomic --coverprofile=coverage.out -p -r ./pkg ./internal ./cmd
     SAVE ARTIFACT coverage.out AS LOCAL coverage.out
 
+OSRELEASE:
+    COMMAND
+    ARG OS_ID
+    ARG OS_NAME
+    ARG OS_REPO
+    ARG OS_VERSION
+    ARG OS_LABEL
+    ENV OS_ID=$OS_ID
+    ENV OS_NAME=$OS_NAME
+    ENV OS_REPO=$OS_REPO
+    ENV OS_VERSION=$OS_VERSION
+    ENV OS_LABEL=$OS_LABEL
+
+    # update OS-release file
+    RUN envsubst >/etc/os-release </usr/lib/os-release.tmpl && \
+        rm /usr/lib/os-release.tmpl
```

Comment:
Could we leave the template for images that extend core and lite.  It would make sense for them to override theses values.

Reply:
right :+1:

Reply:
done in https://github.com/c3os-io/c3os/pull/56/commits/b5d4b1bfd99c288aa55548b0885161d560d3d0da